src/main/scala/piconot/internal/Empty.scala

```diff
@@ -0,0 +1,15 @@
+import DSL_API._
+import picolib._
+import picolib.maze._
+import picolib.semantics._
+import java.io.File
+
+val rulesListEmpty = Robot("start go_left until wall EGTR1", 
+                            "first go_up until wall then go_down EGTR2", 
+                            "second go_down until wall then go_right EGTR3", 
+                            "third go_up until wall then go_right EGTR2"
+                        )
```

Comment:
I do agree with your evaluation that this way of writing the rules doesn't seem incredibly clean. I wonder if it'd help to just have all of the rule strings in a text file by itself, and have sbt run automatically convert the text into parameters for the robot?